.github/workflows/webviz-subsurface-components.yml

```
@@ -19,7 +19,16 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        include:
+          - python-version: 3.6
+            run-performance-tests: false
+          - python-version: 3.7
+            run-performance-tests: false
+          - python-version: 3.8
+            run-performance-tests: false
+          - python-version: 3.9
+            run-performance-tests: true
```

```suggestion
          - python-version: 3.9
            run-performance-tests: false
          - python-version: 3.9  # This job is allowed to fail
            run-performance-tests: true
```
We want the standard Python 3.9 job to be required, but the performance test run to allow to fail (when that is true). One hacky solution from my side is to do 3.9 twice. We could look into later if there are better ways of defining the job matrix GitHub action workflow, but for now I think this is good enough.

---

Oh! I am sorry!... I completely missed it . You had already mentioned that in the previous comment to this but I missed it. I have added another commit